### ISSUE TITLE
cya mydata - multiple source directories

### ISSUE THREAD

**Issue by cspaniard** [cleverwise/cya#11]
The `mydata` command and profiles are a great addition to `cya`.

It would be even more flexible if you could define multiple sources for the same profile.

For example:
```
MYDATA_johnfiles="/home/john/Downloads/ /home/john/Documents/ /mnt/wd-password/john/"
```

This syntax would be easy to understand since `rsync` works the same way. All paths (except last one) would be copied to "destination" (last one).

Needless to say that I would keep it with full paths (like in the example), so source paths do not have to share parent folder.

We do have configuration files in multiple directories in our PBX that could benefit from this enhancement.

That would be very flexible and powerful.

**Comment by cleverwise** [comment]
Yeah I thought about multiple source directories but decided against it.  However I'll revisit that choice based on a few reasons.  I figured just setup multiple profiles and run them.

mydata johndocs
mydata johndls

**Comment by cspaniard** [comment]
Since profiles in general tend to group things, settings, jobs, etc... It feels strange that to do one job you would need more than one profile. As a user, you will need to remember that 2 or 3 profiles always have to be executed together to do the entire job.

The workaround feels backwards... but that is just me... ;)

**Comment by cleverwise** [comment]
True.  I did consider both when originally coding this function.  I'll take another look.

**Comment by cleverwise** [comment]
Included in 2.3.4+

**Comment by cspaniard** [comment]
`Ver 2.3.4`
Although I agree with the documentation that this is a extremely powerful option, I cannot get it work.
It does not give any errors. I does not seem to do anything.

I checked my syntax 3 times and it seems to be correct according to documentation. I must be doing something wrong, but I don't know what else to check.

**Comment by cleverwise** [comment]
I would need to see what you are including.

**Comment by cspaniard** [comment]
```
MYDATA_johnfiles="/home/dsanroma/Descargas/Vuelo/Malaga/:Vuelo;/home/dsanroma/Descargas/Vuelo/Aeropuertos/:info /mnt/"
 ```

**Comment by cleverwise** [comment]
Okay well your backup will need to have /mnt/Vuelo/ and /mnt/info/ thus do they exist?

CYA will **not** create those directories in case those are external mounts.  So your manual backup locations need to created by you.

**Comment by cspaniard** [comment]
I see... makes sense.... I did not see that explanation in the documentation... is it there ? should it be ? ;)
